extensions/wikia/AchievementsII/AchConfig.class.php

```diff
@@ -51,63 +51,62 @@ class AchConfig {
 	private $mNotInTrackStatic;
 
 
-
 	public function __construct() {
 		//$this->mLoadedData = array();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS125)